.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,42 @@
+name: GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: ./client
+
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Since we only want to deploy once, maybe just for cleanliness don't use a matrix here?

Reply:
Makes sense will remove it